docs/source/noaa_csp.rst

```diff
@@ -40,160 +51,252 @@ Login to the NOAA CSP
 
 Log in to the `NOAA CSP <http://noaa.parallel.works/login>`_ and into
 the resources configuration. The user should arrive at the following
-screen.
+screen. Click the "blue" box pointed by the red arrow to login.
+
+.. image:: _static/noaacsp_login_1.png
 
-Note that the ``Username or email`` query is case-sensitive. The user
-will then be prompted for their respective RSA token key using the
-same application use for the other RDHPCS machines (i.e., Hera, Jet,
-etc.,).
+Fill the ``Username / Email`` box with your own username or NOAA-email.
+Note that the ``Username or email`` query is case-sensitive.
+And then the respective RSA token key using the same application used
+for the other RDHPCS machines (i.e., Hera, Jet, etc.,).
 
-.. image:: _static/noaacsp_login.png
+.. image:: _static/noaacsp_login_2.png
 
 *******************************
 Configure the NOAA CSP Instance
 *******************************
 
-Once logged into the NOAA CSP, navigate to the :red-text:`RESOURCES` section
-and click the ``+ Add Resource`` button in the upper-right corner as
-illustrated below.
+Once logged into the NOAA CSP, navigate to the :red-text:`Marketplace` section
+at the left panel as pointed by the red arrow, and click.
+Scrow down to selecet "AWS EPIC Wei CentOS" as in red-circle.
+Note, current global-workflow still using CentOS built spack-stack.
+It will be updated to Rocky 8 soon.
```

Comment:
```suggestion
in the left sidebar indicated by the red arrow, and click.
Scrow down to selecet "AWS EPIC Wei CentOS", circled  in red.
Note that the current global-workflow is still using CentOS-built spack-stack,
but it will be updated to Rocky 8 soon.
```